actors/market/src/lib.rs

```diff
@@ -60,6 +60,9 @@ fil_actors_runtime::wasm_trampoline!(Actor);
 
 pub const NO_ALLOCATION_ID: u64 = 0;
 
+// An exit code indicating that information about a past deal is no longer available.
+pub const EX_DEAL_EXPIRED: ExitCode = ExitCode::new(32);
```

Comment:
Should we not have a `FIRST_ACTOR_SPECIFIC_EXIT_CODE` somewhere, and define this in terms of that?

Reply:
Probably, but will defer